### PR TITLE
Bat Team documentation

### PR DESCRIPTION
**Pull request by pkarman**
Describe the Bat Team practice.

### CODE REVIEW

File: docs/bat-team.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+for the next team.
+
+## Protocols
+
+### Sentry and Slack
+
+Each Bat Team member should take responsibility for one or more Slack channels and monitor for Sentry alerts. These
+come from the production Caseflow system.
+
+When a new Sentry alert appears in Slack, it should be investigated asap. If you cannot investigate it immediately,
```
Comment (pkarman) [inline docs/bat-team.md]: @lomaxap I added this directly in response to what we discussed at retro. What do you think?

File: docs/bat-team.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+emoji tag it with the :bat: emoji.
+
+If a Github ticket already exists for the underlying issue, the Sentry alert should be ignored for a week.
```
Comment (leikkisa) [inline docs/bat-team.md]: Once we know how, it'd be good to document how to determine whether a Github ticket does already exist (e.g. Sentry comment or linking, or some way to label GH tickets to make it more apparent that they are a response to a Sentry alert).

File: docs/bat-team.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+If a Github ticket does not yet exist, create a Github ticket, with a link to the Sentry incident
+in the ticket description. The key evaluation is whether this incident reflects an immediate production issue,
+particularly affecting data integrity, or whether it can be picked up during normal sprint planning.
+The Bat Team should do just enough investigation to determine further action.
```
Comment (leikkisa) [inline docs/bat-team.md]: If it's an immediate production issue, is the expectation that the batteam works on it per line 9? (or elevate it to the right folks?)

Comment (pkarman) [reply docs/bat-team.md]: added some comments about escalation.

**Pull request merged**